### PR TITLE
Create models Coder, Level, Link

### PR DESCRIPTION
**Pull request by OleksandrHavron**

### CODE REVIEW

File: codeArena/user/models.py [hunk 1]
```diff
@@ -24,3 +24,28 @@ def __str__(self):
...
+    phone_number = models.CharField(max_length=15)
+    level_id = models.ForeignKey(Level, on_delete=models.CASCADE)
+    
+    def __str__(self):
+        return f'{self.id}, {self.city}, {self.description}, {self.phone_number}, {self.level_id}'
+
+
+class Link(models.Model):
+    coder_id = models.ForeignKey(Coder, on_delete=models.CASCADE)
+    hyperlink = models.URLField()
```
Comment (vvelizariy) [inline codeArena/user/models.py]: using `url` field name  instead of `hyperlink` would be more appropriate

Comment (vvelizariy) [reply codeArena/user/models.py]: also what sense of moving links to separate table, links can be duplicate for different users?

Comment (OleksandrHavron) [reply codeArena/user/models.py]: Ok.

Comment (OleksandrHavron) [reply codeArena/user/models.py]: It's links for socials, there can be more than one link for one user.

**Review by s0fko — APPROVED** [review]

**Review by Viktorika-p — APPROVED** [review]

**Review by MetalMari — APPROVED** [review]

**Pull request merged**